website/members/models/profile.py

```diff
@@ -245,6 +245,17 @@ class Profile(models.Model):
         default=True,
     )
 
+    # Begin Lisanne
+    photo_permissions = models.BooleanField(
+        verbose_name=_("Opt-in social media pictures"),
+        help_text=_(
+            "Consent to all your photos being shared across social media platforms."
+        ),
+        blank=False,
+        null=True,
+    )
```

Comment:
Have you considered that this would allow access to the full user profile for the Paparazcie unless we make this a part of some special page? I already mentioned this in the issue because that would be difficult to do.

It's probably easier to create a allowlist model like the blacklist model we have for payments: https://github.com/svthalia/concrexit/blob/master/website/payments/models.py#L111. Then the paparazcie can have view permissions and always see this info.

The only 'hard' part there is making it possible for users to change their preferences thus requiring a larger profile edit page with a part that is in an extra form field.